src/components/About.js

```diff
@@ -48,11 +68,15 @@ export default function About(props) {
         </div>
 
         <div className={imgStyles.imgWrapper}>
-          <div
-            role="img"
-            aria-label="blonde woman in black and white outfit"
-            className={imgStyles.img}
-          ></div>
+          {showImage ? (
+            <div
+              role="img"
+              aria-label="blonde woman in black and white outfit"
```

Comment:
What lead you to use role img and aria label here? Why not just using a normal `<img>`?

Reply:
It has to do with images being set on background. It was supposed to solve the <img> not having a src and the black border the pictures had as a result.